NMS-8861: Add information about logging (#4550)
OpenNMS/opennms: docs/modules/reference/pages/daemons/daemon-config-files/perspectivepollerd.adoc

[[ref-daemon-config-files-perspectivepoller]]
= PerspectivePoller

> Handles xref:operation:application-perspective-monitoring/introduction.adoc[application perspective monitoring].

Reviewer: ```suggestion
Queries aggregated percentage availability for a given specific application and its services, as part of the xref:operation/application-perspective-monitoring/application-perspective-monitoring.adoc[application perspective monitoring].
```

> Queries aggregated percentage availability for a given specific application and its services, as part of the xref:operation/application-perspective-monitoring/introduction.adoc[application perspective monitoring].

Reviewer: operation:application-perspective-monitoring/introduction.html

> Queries aggregated percentage availability for a given specific application and its services, as part of the xref:operation/application-perspective-monitoring/introduction.adoc[application perspective monitoring].

Reviewer: ```suggestion
Queries aggregated percentage availability for a given specific application and its services, as part of the xref:operation:application-perspective-monitoring/introduction.adoc[application perspective monitoring].
```

> Queries aggregated percentage availability for a given specific application and its services, as part of the xref:operation/application-perspective-monitoring/application-perspective-monitoring.adoc[application perspective monitoring].

Reviewer: ```suggestion
Queries aggregated percentage availability for a given specific application and its services, as part of the xref:operation/application-perspective-monitoring/introduction.adoc[application perspective monitoring].
```
